packages/vuepress-theme-odyssey/templates/DocsTemplateHome.vue

```diff
@@ -4,12 +4,24 @@
     <header class="docs-hero">
       <div class="docs-hero--content">
         <h1 class="docs-hero--title">{{ $page.frontmatter.headline }}</h1>
-        <p  class="docs-hero--desc">{{ $page.frontmatter.lead }}</p>
+        <p class="docs-hero--desc">{{ $page.frontmatter.lead }}</p>
       </div>
-      <div class="docs-hero-moon" v-html="require(`!html-loader!../../docs/.vuepress/public/images/home-hero-moon.svg`)"></div>
-      <div class="docs-hero-rocket" v-html="require(`!html-loader!../../docs/.vuepress/public/images/home-hero-rocket.svg`)"></div>
+      <!-- eslint-disable vue/no-v-html -->
+      <div
+        class="docs-hero-moon"
+        v-html="
+          require(`!html-loader!../../docs/.vuepress/public/images/home-hero-moon.svg`)
```

Comment:
Are there alternatives to using the html-loader? Would be nice to not have to add `eslint-disable` in all these files.

Reply:
I 100% agree, I will log an issue to investigate and replace these.

Reply:
Logged #809